src/main/java/com/minecolonies/rankup/util/PermissionsUtils.java

```diff
@@ -95,25 +92,21 @@ public String getPlayerHighestRankingGroup(User user)
 
     public Integer timeToNextGroup(final User user)
     {
-        final GroupsConfig groupsConfig = (GroupsConfig) plugin.getAllConfigs().get(GroupsConfig.class);
-        final AccountConfigData playerData = (AccountConfigData) plugin.getAllConfigs().get(AccountConfigData.class);
-
-        final AccountConfigData.PlayerConfig playerConfig = playerData.playerData.get(user.getUniqueId());
+        final GroupsConfig groupsConfig = plugin.configUtils.getGroupsConfig(user.getPlayer().orElse(null));
 
-        final String currentGroup = getPlayerHighestRankingGroup(user);
-        final String nextGroup = getNextGroup(groupsConfig.groups.get(currentGroup).rank);
+        final String nextGroup = getNextGroup(user);
 
         if (nextGroup.equals(""))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the "" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1132)